client/my-sites/upgrades/domain-management/email/index.jsx

```diff
@@ -41,8 +43,14 @@ const Email = React.createClass( {
 	},
 
 	render() {
+		const showPlanFeatures = config.isEnabled( 'manage/plan-features' );
+
+		const classes = classNames( 'domain-management-email', {
+			'is-wide-layout': showPlanFeatures
```

Comment:
Until @mtias chimes in, let's assume the latter, that we pass in a prop to main, and this adds an additional class. @lamosty does the `margin-top` increase or decrease with `has-personal-plan`? We might want to pick a prop name that is more descriptive of what the main component does when the prop is set.